File: php/src/FizzBuzz.php
<?php
namespace App;

class FizzBuzz
{
    public function run(int $num): string
    {
        $str = ($num % 3 === 0) ? 'Fizz': '';
        $str .= ($num % 5 === 0) ? 'Buzz': '';
        return ($str) ? $str: (string)$num;
    }
}

File: golang/fizzbuzz/fizzbuzz.go
package fizzbuzz

import "strconv"

func fizzbuzz(num int) string {
	if num%15 == 0 {
		return "FizzBuzz"
	}
	if num%3 == 0 {
		return "Fizz"
	}
	if num%5 == 0 {
		return "Buzz"
	}
	return strconv.Itoa(num)
}
